packages/block-library/src/image/index.php

```diff
@@ -292,19 +337,30 @@ class="wp-lightbox-overlay zoom"
 			data-wp-bind--style="state.overlayStyles"
 			tabindex="-1"
 			>
-				<button type="button" aria-label="$close_button_label" style="fill: $close_button_color" class="close-button">
-					<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" width="20" height="20" aria-hidden="true" focusable="false"><path d="m13.06 12 6.47-6.47-1.06-1.06L12 10.94 5.53 4.47 4.47 5.53 10.94 12l-6.47 6.47 1.06 1.06L12 13.06l6.47 6.47 1.06-1.06L13.06 12Z"></path></svg>
+				<button type="button" aria-label="$close_button_label" style="fill: $close_button_color" class="wp-lightbox-close-button">
```

Comment:
Any construction that leaves an empty `aria-label` value is a potential problem; if the button has an empty `aria-label`, that could cause assistive technology to interpret the button as unlabeled. 

They should be smart enough to not do that, but it would be better to remove the `aria-label` value entirely if the text label is empty.

Also applies in the prev/next buttons.